app/views/providers/gateway_evidences/_uploaded_files.html.erb

```diff
@@ -14,8 +10,6 @@ success_message = success_message
   </thead>
   <tbody class="govuk-table__body">
 
-  <span alert-live="assertive" class="govuk-visually-hidden"><%= success_message %></span>
```

Comment:
ARIA live regions are generally used for accessibility purposes, are you sure this no longer required?
If required, it is best to move it to elsewhere in DOM as this isn't a valid location.

Reply:
There is no scenario where `success_message` is anything other than `nil`, so I'm not sure what use an empty and invisible (except to screen readers) span is?

@RoseSAK do you have any idea why these were here in the first place?

Reply:
I don't think these are needed at all.